src/Symfony/Component/DependencyInjection/Tests/EnvVarProcessorTest.php

```diff
@@ -420,4 +420,30 @@ public function testGetEnvKeyChained()
             );
         }));
     }
+
+    /**
+     * @dataProvider validNullables
+     */
+    public function testGetEnvNullable($value, $processed)
+    {
+        $processor = new EnvVarProcessor(new Container());
+
+        $result = $processor->getEnv('nullable', 'foo', function ($name) use ($value) {
+            $this->assertSame('foo', $name);
+
+            return $value;
+        });
+
+        $this->assertSame($processed, $result);
+    }
+
+    public function validNullables()
+    {
+        return array(
+            array('hello', 'hello'),
+            array('', null),
+            array('null', null),
+            array('NULL', null),
```

Comment:
You should add testcase `array('Null', null)` as you mentioned it above

Reply:
Good catch

Reply:
Besides, I wonder if adding `~` as nullable would be relevant. For the moment I don't think so.

Reply:
> Besides, I wonder if adding ~ as nullable would be relevant. For the moment I don't think so.

I am against it, it belongs to YAML only AFAIK